Sources/EdgeNetworkHandlers/RequestBuilder.swift

```diff
@@ -26,6 +26,9 @@ class RequestBuilder {
     /// XDM payloads to be attached to the request
     var xdmPayloads: [String: AnyCodable] = [:]
 
+    var sdkConfig: SDKConfig?
```

Comment:
Can you add a description to the property? Such as "SDK configuration metadata containing original datastream ID if overridden."